termsaver/__init__.py

```diff
@@ -165,11 +184,19 @@ def getScreen():
         parsers = {}
         screenparsers = parser.add_subparsers()
         screen = None
+        random_screen_name = None
+        if use_random is True:
+            random_screen_name = random.choice(available_random_screens)
+
         for s in get_available_screens():
             # Create the parsers for each screen.
             parsers[s.__name__.lower()] = screenparsers.add_parser(s.__name__.lower(), formatter_class=SmartFormatter, conflict_handler='resolve')
-            if s().name.lower() == args.screen:
-                screen = s
+            if use_random is False:
```

Comment:
in line 189 you could just choose to override the args.screen value so then you don't need to change anything in the code below (confirming again the random argument within the available screens loop)... what do you think?